data/cameras.xml

```diff
@@ -9976,9 +9976,6 @@
 	<Camera make="RICOH IMAGING COMPANY, LTD." model="GR" mode="dng">
 		<ID make="Ricoh" model="GR">GR</ID>
 	</Camera>
-	<Camera make="RICOH" model="GR DIGITAL 2" mode="dng">
-		<ID make="Ricoh" model="GR II">Ricoh GR DIGITAL 2</ID>
```

Comment:
Presumably this should be `<ID make="Ricoh" model="GR DIGITAL 2">Ricoh GR DIGITAL 2</ID>`?

Reply:
I think it's ok to remove completely - AFAICT rawspeed's DngDecoder should get "RICOH"/"GR DIGITAL 2" into canonical_make/model, and "Ricoh GR DIGITAL 2" into canonical_id from the DNG UniqueCameraModel tag without the help of cameras.xml in this case, as that DNG tag exists and is correct.

Reply:
I agree with @kmilos. Besides, if I remember correctly, there were 4 versions of GR DIGITAL cameras, so, if they need to be added, it would be best to add all models.